Makefile

```diff
@@ -3,7 +3,7 @@ PACKAGE_NAME := $(shell awk '/"name":/ {gsub(/[",]/, "", $$2); print $$2}' packa
 RPM_NAME := cockpit-$(PACKAGE_NAME)
 VERSION := $(shell T=$$(git describe 2>/dev/null) || T=1; echo $$T | tr '-' '.')
 ifeq ($(TEST_OS),)
-TEST_OS = fedora-36
+TEST_OS = fedora-37
```

Comment:
`HACKING.md` still mentions `fedora-36` btw.

Reply:
Thanks for spotting! Fixed.